src/components/ChatLog.js

```diff
@@ -0,0 +1,41 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ChatLog = (props) =>{
+    const chatMessagesData = props.entries.map(message => {  
+        return (
+        <div key={message.id}>
+            <ChatEntry
+            id={message.id}
+            sender={message.sender}
+            body={message.body}
+            timeStamp={message.timeStamp}
+            liked={message.liked}
+            onUpdateChat={props.onUpdateChat}
+            />
```

Comment:
So you see how setting all of those props took up a lot of lines of code? We could make that more concise with this here: 
```javascript
 < ChatEntry onUpdateChat={props.onUpdateChat} key={message.id} entry={message} />
```
Then in each `chatEntry` we could access any prop we needed with `props.entry.myProp`